src/main.c

```diff
@@ -76,12 +74,13 @@ int main(int argc, char *argv[]) {
     struct arg_lit *vers = arg_lit0("v", "version", "print version information and exit");
     struct arg_lit *listen = arg_lit0("l", "listen", "Start a network game server");
     struct arg_str *connect = arg_str0("c", "connect", "<host>", "Connect to a remote game");
+    struct arg_str *renderer = arg_str0("X", "renderer", "<renderer>", "Renderer to use (default: opengl3)");
```

Comment:
Could you name this "force_renderer" or similar, to communicate that this is not the primary way of setting the renderer?
Same for the engine_init struct.

Reply:
I don't think this needs a one-letter shorthand either, feel free to replace "X" (which allows -X) with NULL

Reply:
Please do not specify the default as opengl3 here, that may not be true in the future